rpc/proto/mayastor.proto

```diff
@@ -292,6 +293,21 @@ message RebuildStateReply {
   string state = 1; // current rebuild state (i.e. ready/running/completed etc.)
 }
 
+message RebuildStatsRequest {
+  string uuid = 1;  // uuid of the nexus
+  string uri = 2;   // uri of the destination child
+}
+
+message RebuildStatsReply {
+  uint64 blocks_total = 1; // total number of blocks to recover
+  uint64 blocks_recovered = 2; // number of blocks recovered
+  uint64 progress = 3; // rebuild progress in % (0-100)
```

Comment:
nit - do you need "(0-100)" on the end? It's a % so it's implied

Reply:
Done, also fixed the same comment from rebuild_api